lang/rust/rust-values.mk

```diff
@@ -5,7 +5,7 @@
 # Rust Environmental Vars
 RUSTC_HOST_SUFFIX:=$(word 4, $(subst -, ,$(GNU_HOST_NAME)))
 RUSTC_HOST_ARCH:=$(HOST_ARCH)-unknown-linux-$(RUSTC_HOST_SUFFIX)
-CARGO_HOME:=$(STAGING_DIR)/host/cargo
+CARGO_HOME:=$(DL_DIR)/cargo
```

Comment:
Mhhh... I don't like bloating the DL_DIR. From documentation the CARGO_HOME can also have cached bin and other configuration file. DL_DIR should really only have downloaded stuff.

Maybe a better place would be TMP_DIR?

Also consider that an user might use the same buildroot to compile the different ARCH. We should make sure if that might cause problems with using the same CARGO_HOME directory for this case...

Reply:
> From documentation the CARGO_HOME can also have cached bin and other configuration file. DL_DIR should really only have downloaded stuff.

The bin directory holds binaries the user has built, where under normal circumstances `CARGO_HOME` is `~/.cargo`. Neither target or host packages should install binaries there, and no configuration file should be created there, so there is no problem to put `CARGO_HOME` in `$(DL_DIR)`.

> Also consider that an user might use the same buildroot to compile the different ARCH. We should make sure if that might cause problems with using the same CARGO_HOME directory for this case...

Under normal circumstances `CARGO_HOME` is `~/.cargo`, which would be shared among all of the architectures the user is building, so I would assume they have solved this issue.

Reply:
Ok something to check sometime just in case we miss some option to not use CARGO_HOME. Thanks for the explainations.